84.md

```diff
@@ -0,0 +1,98 @@
+NIP-84
+======
+
+Highlights
+----------
+
+`draft` `optional` `author:pablof7z`
+
+This NIP defines `kind:9802`, a "highlight" event, to signal content a user finds valuable.
+
+## Format
+The `.content` of these events is the highlighted portion of the text.
+
+`.content` might be empty for highlights of non-text based media (e.g. NIP-94 audio/video).
+
+### References
+Events SHOULD tag the source of the highlight, whether nostr-native or not.
+`a` or `e` tags should be used for nostr events and `r` tags for URLs.
+
+When tagging a URL, clients generating these events SHOULD do a best effort of cleaning the URL from trackers
+or obvious non-useful information from the query string.
+
+### Attribution
+Clients MAY include one or more `p` tags, tagging the original authors of the material being highlighted; this is particularly
+useful when highlighting non-nostr content for which the client might be able to get a nostr pubkey somehow
+(e.g. prompting the user or reading a `<meta name="nostr:pubkey" />` tag on the document). A role MAY be included as the
+last value of the tag.
+
+```json
+[ "p", "...", "author" ],
+[ "p", "...", "author" ],
+[ "p", "...", "editor" ],
+```
+
+### Context
+Clients MAY include a `context` tag, useful when the highlight is a subset of a paragraph and displaying the
+surrounding content might be beneficial to give context to the higlight.
```

Comment:
Maybe clarify that this is a larger text excerpt of the original content, that wasn't immediately obvious to me. Or just refer to the example below.

Reply:
Oh, yeah, good point